broker/src/main/java/org/apache/rocketmq/broker/offset/ConsumerOrderInfoManager.java

```diff
@@ -149,6 +149,13 @@ public boolean checkBlock(String topic, String group, int queueId, long invisibl
         return orderInfo.needBlock(invisibleTime);
     }
 
+    public void clearBlock(String topic, String group, int queueId) {
+        table.computeIfPresent(buildKey(topic, group), (key, val) -> {
```

Comment:
How about using org.apache.rocketmq.common.utils.ConcurrentHashMapUtils#computeIfAbsent?

Reply:
@RongtongJin How can computeIfAbsent helps computeIfPresent?

Reply:
Sorry,I misread it